bin/lambda-cfn.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+var cli = require('../lib/cli');
+var d3 = require('d3-queue');
+
+var parsed;
+try { parsed = cli.parse(process.argv.slice(2), process.env); }
+catch (err) { return finished(err); }
+
+// if (parsed.options.all) {
+  // call a helper function to compile the list of
+  // stacks for the command to loop over
+  //var q = d3.queue(1);
+  //for (var stack in parsed.options.stacks) {
+    // modify parsed and pass to command loop
+    // q.defer(cli.main, modified_parsed)
+    // q.await(function(err,data) {});
+//  }
+// } else {
+  cli.main(parsed, finished);
+
+  function finished(err, data) {
+    if (err) process.stderr.write(err.message + '\n');
```

Comment:
I seem to remember that one of the linters complained about/forbid using `console.log`, but this code came directly from `cfn-config` so I don't actually know.